docs/auto-instrumentation-install.md

```diff
@@ -403,6 +410,40 @@ provides best effort support with issues related to native OpenTelemetry instrum
 | apache-httpd            | OpenTelemetry | Available   | Needs Validation |                              | [Link](https://github.com/open-telemetry/opentelemetry-apache-httpd-instrumentation) | ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-apache-httpd |
 | nginx                   | OpenTelemetry | Available   | Needs Validation |                              | [Link](https://github.com/open-telemetry/opentelemetry-apache-httpd-instrumentation) | ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-apache-httpd |
 
+### CRD Management
+
+When deploying the operator, the required Custom Resource Definitions (CRDs) must be deployed beforehand.
+
+#### Recommended Approach: Automated CRD Deployment
+
+Set the Helm chart value `operatorcrds.install=true` to allow the chart to handle CRD deployment automatically.
+_This option deploys the CRDs using a local subchart, available at https://github.com/signalfx/splunk-otel-collector-chart/tree/main/helm-charts/splunk-otel-collector/charts/opentelemetry-operator-crds._
+
+#### Alternative Approach: Manual CRD Deployment
+
+If you prefer to manage CRD deployment manually, apply the CRDs using the commands below before installing the Helm chart:
+
+```bash
+curl -sL https://raw.githubusercontent.com/open-telemetry/opentelemetry-operator/main/config/crd/bases/opentelemetry.io_opentelemetrycollectors.yaml | kubectl apply -f -
+curl -sL https://raw.githubusercontent.com/open-telemetry/opentelemetry-operator/main/config/crd/bases/opentelemetry.io_opampbridges.yaml | kubectl apply -f -
+curl -sL https://raw.githubusercontent.com/open-telemetry/opentelemetry-operator/main/config/crd/bases/opentelemetry.io_instrumentations.yaml | kubectl apply -f -
+```
+
+#### CRD Updates
+
+With Helm v3.0 and later, CRDs created by this chart are not updated automatically. To update CRDs, you must apply the updated CRD definitions manually.
+Refer to the [Helm Documentation on CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/) for more details.
+
+#### CRD Cleanup
+
+When uninstalling this chart, the OpenTelemetry CRDs are not removed automatically. To delete them manually, use the following commands:
```

Comment:
Same as previous comment, not blocking: Is there any way to ensure these commands stay up to date and work properly?

Reply:
I want to update our CI/CD to download the the latest/current CRDs whenever we the operator updates and validate CRDs are proper and commands work. That would be good enough for now until upstream starts using versioning more.